pkg/options/options.go

```diff
@@ -82,7 +82,10 @@ func newPullPolicyValue(defaultValue v1.PullPolicy, v *v1.PullPolicy) *pullpolic
 }
 
 const (
-	defaultMysqlImage    = "percona:5.7"
+	// because of moving percona docker images to centos they broke the mysql
```

Comment:
hi @AMecea you can send feedback directly to me if I can fix something for you in upstream

Reply:
Hi @delgod, I wanted to contact you, we can chat further on the k8s slack if you want. The problem with the new percona images based on centos is that you configure mysql from `/etc/my.cnf` insted of `/etc/mysql/my.cnf` (as debian based images did) and the operator mounts custom configs in `/etc/mysql` but for some reason mounting the config map to `/etc/mysql` mounts it to `/etc` and brakes everything.

Try running:
```shell
➜  test docker run --rm -v $(pwd)/mysql:/etc/mysql/ -it percona:5.7 /bin/bash                                                                                                                                       
bash-4.2$ ls -la /etc/
total 28
drwxr-xr-x 3 1000 1000 4096 Nov 28 11:06 .
drwxr-xr-x 1    0    0 4096 Nov 28 11:12 ..
drwxr-xr-x 2 1000 1000 4096 Nov 28 11:01 conf.d
-rw-r--r-- 1    0    0   13 Nov 28 11:12 hostname
-rw-r--r-- 1    0    0  174 Nov 28 11:12 hosts
-rw-r--r-- 1 1000 1000   37 Nov 28 11:01 my.cnf
-rw-r--r-- 1    0    0   53 Nov 28 11:12 resolv.conf
bash-4.2$ 

```